network/dns-cloud/how-to/delete-external-domain.mdx

```diff
@@ -0,0 +1,46 @@
+---
+meta:
+  title: How to delete a domain from Scaleway Domains & DNS
+  description: This page explains how to delete a domain from Scaleway Domains & DNS
+content:
+  h1: How to delete a domain from Scaleway Domains & DNS
+  paragraph: This page explains how to delete a domain from Scaleway Domains & DNS
+dates:
+  validation: 2021-12-31
+  posted: 2021-06-07
+---
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
+- You have [linked your domain](/network/dns-cloud/how-to/add-external-domain) with Scaleway Domains & DNS
+
+</Message>
+
+1. Click **DNS** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. 
+
+ The DNS zone dashboard displays.
+
+2. Click <Icon name="more" />, then **More info** next to the domain you want to delete.
+
+3. Click **Delete domain**. 
+ 
+ A pop-up displays. 
+
+4. Type **DELETE** to confirm the deletion of the domain, then click **Delete this domain**.  
+    <Lightbox src="scaleway-dns-delete-domain.webp" alt="" />
+    <Message type="important">
+
+    This will permanently delete your external domain and its related zones from Scaleway Domains & DNS. This action is irreversible and your domain name may become unresolvable. 
```

Comment:
```suggestion
    This will permanently delete your external domain and its related zones from Scaleway Domains and DNS. This action is irreversible and your domain name may become unresolvable. 
```